cos_registration_server/devices/views.py

```diff
@@ -62,10 +62,12 @@ def __init__(
 
     cos_model_name = settings.COS_MODEL_NAME
 
+    # fmt: off
```

Comment:
Why do you need that?

Reply:
it keeps messing up that link to this weird: `grafana_main_link = f"{base_url}/{cos_model_name}-grafana/" f"dashboards/"`